lib/einhorn/command.rb

```diff
@@ -138,7 +145,7 @@ def self.signal_all(signal, children=nil, record=true)
         end
       end
 
-      "Successfully sent #{signal}s to #{signaled.length} processes: #{signaled.inspect}"
+      "Successfully sent #{signal}s to #{signaled.length} processes: #{signaled.keys}"
```

Comment:
We should drop support for 1.8. But until we do, we should make this `signaled.keys.inspect` since `Array#to_s` in 1.8 is just `.join('')`